force-app/main/default/classes/HH_Container_LCTRL.cls

```diff
@@ -514,12 +536,16 @@ public with sharing class HH_Container_LCTRL {
 
     @AuraEnabled
     public static Map<String, String> getFieldLabels() {
+        String undeliverableAddressLabel = canReadContactUndeliverable ?
```

Comment:
Don't we need namespace here as well?

Reply:
In this case, we're stripping the namespace in the Aura component anyways, so this should work fine.